pallets/did/src/mock.rs

```diff
@@ -386,6 +394,16 @@ pub(crate) fn get_none_key_call() -> RuntimeCall {
 	})
 }
 
+#[cfg(not(feature = "runtime-benchmarks"))]
+pub(crate) fn build_test_origin(account: AccountId32, did: AccountId32) -> RuntimeOrigin {
```

Comment:
This should work as well. `DidIdentifier` and `AccountId` are hard coded in lines 55-70.

```suggestion
pub(crate) fn build_test_origin(account: AccountId, did: DidIdentifier) -> RuntimeOrigin {
```